source/codegen/metadata/nirfmxpulse/config.py

```diff
@@ -27,7 +27,7 @@
         "NIComplexSingle": "nidevice_grpc.NIComplexNumberF32",
         "NIComplexDouble": "nidevice_grpc.NIComplexNumber",
     },
-    'code_readiness': 'NextRelease',
+    'code_readiness': 'Release',
```

Comment:
Work with @ThangamV-NI to make sure that testing is sufficient for Release.

Reply:
Can you add session & driver-api tests, similar to other standards?

Reply:
I have added those test cases as part of the PR: #1111 by referring VNA personality
You can check the description for the successful test runs.